hash/dc.xml

```diff
@@ -528,15 +531,16 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 		<info name="alt_title" value="4x4 Evolution"/>
 		<info name="ring_code" value="%T41903N Z0A21C"/>
 		<info name="barcode" value="6 63593 12014 4"/>
+		<info name="redump_name" value="4x4 Evo (USA)"/>
```

Comment:
What if redump mantainers starts changing titles left and right. 
This node has no sense as a data standpoint.

Reply:
This is what the equivalent redump DAT entry looks like:
```
	<game name="4x4 Evo (USA)">
		<category>Games</category>
		<description>4x4 Evo (USA)</description>
		<rom name="4x4 Evo (USA).cue" size="1205" crc="73a133eb" md5="e01ab68d00a0b4c59633324894ebae3e" sha1="30ba7f32f0a6b7021e9f027eeb9a1f1e0081f730"/>
		<rom name="4x4 Evo (USA) (Track 01).bin" size="1425312" crc="2ebac441" md5="725de5c63f7afdb715d3d77f5def9960" sha1="ece1a9f45f34f2673a109024081e0fa1dcb1f8db"/>
		<rom name="4x4 Evo (USA) (Track 02).bin" size="1589952" crc="8557fcaa" md5="824ff123fbce45b5883962eb3fa43cab" sha1="a3705cb0ce53c7adf906c5af76887406984de6a9"/>
		<rom name="4x4 Evo (USA) (Track 03).bin" size="768718272" crc="8561954d" md5="8aede0810539c3244fed0ee51bc35c91" sha1="67825d9fd1a48954e49eb568fa97915d25c55cb2"/>
		<rom name="4x4 Evo (USA) (Track 04).bin" size="40675488" crc="77d32b88" md5="d3c57643818bd25aaa571686c6b8e2d6" sha1="c641b8691e273a20ec7cfae943035fd6ef488c2c"/>
		<rom name="4x4 Evo (USA) (Track 05).bin" size="33123216" crc="d0fad42b" md5="58b1e472a8017430b631de20428bea3a" sha1="37d0adb73d216bd371fb192b56af695ecbd29734"/>
		<rom name="4x4 Evo (USA) (Track 06).bin" size="37977744" crc="c3f98353" md5="f116a56829c67c6d215fb0fe9e0927c5" sha1="8731d73b1f3d1d24ac3ecd12083d39371af7899c"/>
		<rom name="4x4 Evo (USA) (Track 07).bin" size="29409408" crc="6b99634d" md5="d3b4e2a3294b263de4c3d623a57b3d90" sha1="9b8b05cd0334d6f5a82a2a64987cc055c0db998e"/>
		<rom name="4x4 Evo (USA) (Track 08).bin" size="26626992" crc="f4e66651" md5="ec561a0e3170ab8d87a40c9e664c31e0" sha1="ccea687610772c72b6b7cb0c45feae272b036ba4"/>
		<rom name="4x4 Evo (USA) (Track 09).bin" size="27850032" crc="01ba5888" md5="8da91d8091a4af54cd47ccfe94b06e8e" sha1="d9d1b36cf11a7db47270719b2a1e25793ff19815"/>
		<rom name="4x4 Evo (USA) (Track 10).bin" size="34847232" crc="41322d91" md5="c0047f37c0e71e64893ecb1c4c727b4b" sha1="3220b45caf9e049151fcb87c76a6281dc092bfc4"/>
		<rom name="4x4 Evo (USA) (Track 11).bin" size="186532416" crc="4b05a5ed" md5="47674b01da5a6d47cba61b02f9c41c02" sha1="23daaafe4d5e5d0a10854bea0d610df4b67fb0ef"/>
	</game>
```

The redump game tag's name value is the unique key in their DAT, and that's all I'm trying to reference, outside of the hashes themselves there is no other key. 

Agree if there is a change on their side it would need to be updated in the software list, but I don't see how that is worse than the situation today which has no mapping at all, and that's likely one of the biggest factors behind https://github.com/mamedev/mame/issues/2517 being open since 2017 with little progress.  If the redump team did make changes large changes I anticipate it would be tied to an updated naming convention and it wouldn't be terribly difficult to update all the mappings wholesale.  

If you have another suggestion I'm open to try other things.  The other possibility is to just try and use the actual hashes in some way, but those can also change on occasion when dumps get updated, so aren't completely immune to the problem either and add a lot more complexity in general.